receiver/redisreceiver/integration_test.go

```diff
@@ -53,6 +53,7 @@ func TestIntegrationV6(t *testing.T) {
 }
 
 func TestIntegrationV7Cluster(t *testing.T) {
+    t.Skip("Skipping due to flakieness, possibly related to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26194")
```

Comment:
Looks like wrong link

Reply:
This should be an open issue. I don't think the linked issue is strictly relevant